packages/web-forms/src/components/controls/Input/InputGeopointReadonly.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
```

Comment:
I'm not sure about the name of this component. As I understand it, the intent is to have a common presentation of _geopoint value as text_. This lines up with what we discussed, and makes sense! I think that intent might be a better basis for naming the component itself.

**Future consideration:** while it'd probably be overkill (now), it's also worth thinking about whether _every_ `NoteNode<V>` / `InputNode<V>` pair might want a similar shared _value-as-text_ component? That would be good for consistency, and it would also be good for when we might want to display the value of an `InputNode` rather than the control (e.g. to collapse or summarize some/all of the form state).

Reply:
I'm renaming it to `GeopointFormattedValue.vue` and moving it out of /input folder. 

`packages/web-forms/src/components/controls/GeopointFormattedValue.vue`